### PR TITLE
Markdown support in FAQs

### LINKED ISSUE #190: Support for Markdown parsing
#### Desired Behavior
in FAQs...

#### Implementation
react-markdown and/or react-gfm

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by lakshz**
**Notes for Reviewers**

This PR fixes #190 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
light theme issue: 
image

**Comment by vishalvivekm** [comment]
@lakshz will it be possible for you to revert the formating and linting changes ?

File: site/src/assets/data/faq/index.js [hunk 1]
```diff
@@ -15,80 +14,85 @@ const data = {
...
+    //         "The sandbox environment is not connected to an active Kuberentes cluster, and as such, depending upon which Meshery Playgrouns environment you use specific actions within MeshMap Designer are also disabled. Meshery and MeshMap are feature-rich, sophisticated management applications for cloud native infrastructure. To access their full set of capabilities, simply deploy your own copy of Meshery into the environment of your choosing.",
+    //     ],
+    // },
+    {
+      question:
+        "Are designs that I create in the Meshery Playground saved or will I lose any work that I do in the playground?",
+      category: "Meshery Playground",
+      answer: `
+Yes, all of the designs that you have created in the Meshery Playground are saved to your user account and will be available to you each time that you return to the Playground. The designs will also be available in your other Meshery deployments.
```
Comment (vishalvivekm) [inline site/src/assets/data/faq/index.js:66]:
```suggestion
- Yes, all of the designs that you have created in the Meshery Playground are saved to your user account and will be available to you each time that you return to the Playground. The designs will also be available in your other Meshery deployments.
```

Comment (lakshz) [reply site/src/assets/data/faq/index.js]: Done

**Comment by lakshz** [comment]
I've removed some of the unwanted indentation changes. 
Should I create an issue to set up `.prettierrc` on the repo?

File: site/src/components/Faq/index.js [hunk 2]
```diff
@@ -29,50 +32,65 @@ const Faq = (props) => {
...
+                </AccordionBody>
+              </AccordionItem>
+            ))}
+          </>
+        ))}
+      </Accordion>
     </FaqSectionWrapper>
   )
-}
+};
```
Comment (vishalvivekm) [inline site/src/components/Faq/index.js:94]:
```suggestion
}
```

**Comment by vishalvivekm** [comment]
let's wait for deploy log; https://app.netlify.com/sites/meshery-play/deploys/662d0a6cbab8ea0008732a8b

**Comment by leecalcote** [comment]
Syncing with HEAD

**Comment by vishalvivekm** [comment]
building successfully...
https://deploy-preview-191--meshery-play.netlify.app/

**Review by vishalvivekm — APPROVED** [review]
LGTM.
Thanks a bunch @lakshz

**Pull request merged**